style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+}
+
+.body-white {
+  width: 375px;
+  background: #f7f7f9;
+}
+
+.header {
+  width: 23.4375rem;
+  background: white;
+  background-image: url(images/bg-image.png);
+  background-repeat: no-repeat;
+  min-height: 70vh;
+  padding: 5%;
+  border-bottom-left-radius: 70px;
+}
```

Comment:
- [ ] This section is supposed to be aligned with flex box as per the instructions. Kindly add a display of flex box to this container. This gives us a lot of options to align all our elements, plus, it's responsive. 👍🏾